contracts/staking/IApplication.sol

```diff
@@ -19,37 +19,37 @@ pragma solidity 0.8.9;
 /// @notice Generic interface for an application. Application is an external
 ///         smart contract or a set of smart contracts utilizing functionalities
 ///         offered by Threshold Network. Applications authorized for the given
-///         operator are eligible to slash the stake delegated to that operator.
+///         provider are eligible to slash the stake delegated to that provider.
 interface IApplication {
     /// @notice Used by T staking contract to inform the application that the
-    ///         authorized amount for the given operator increased.
+    ///         authorized amount for the given provider increased.
     ///         The application may do any necessary housekeeping.
     function authorizationIncreased(
-        address operator,
+        address stakingprovider,
```

Comment:
Any reason we don't use camel case here (i.e. `stakingProvider`) but we do everywhere else?

Reply:
oh, thank you, I thought I fixed it but apparently missed few places
✔️